projects/storefrontlib/src/shared/components/modal/confirm-modal/confirm-modal.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+import { ModalService } from '../../../../shared/components/modal/modal.service';
+import { ICON_TYPE } from '../../../../cms-components/misc/icon/icon.model';
+
+@Component({
+  selector: 'cx-confirm-modal',
+  templateUrl: './confirm-modal.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ConfirmModalComponent {
```

Comment:
I can't find this component used anywhere. Can we remove it, its module and styles? @Pucek9

Reply:
yes, that's no longer used after introduced toggle-status component

Reply:
i've dropped it